roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -74,6 +74,9 @@ kubernetesMasterConfig:
   masterCount: 1
   masterIP: ""
   podEvictionTimeout: ""
+  proxyClientInfo:
```

Comment:
We also need something to update version fact right after we install packages so that we can use it on first run.